hw/_index.md

```diff
@@ -42,14 +42,14 @@ Finally, we provide the same set of information for all available [top level des
 * [Earl Grey Chip Datasheet]({{< relref "hw/top_earlgrey/doc" >}})
 * [Earl Grey Chip Specification]({{< relref "hw/top_earlgrey/doc/design" >}})
 * [Earl Grey Chip DV Document]({{< relref "hw/top_earlgrey/doc/dv" >}})
-* [DV simulation results, with coverage (nightly)](https://reports.opentitan.org/hw/top_earlgrey/dv/latest/results.html)
-* [Connectivity results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/conn/jaspergold/latest/results.html)
-* [AscentLint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/lint/ascentlint/latest/results.html)
-* [Verilator lint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/lint/verilator/latest/results.html)
-* [Style lint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/lint/veriblelint/latest/results.html)
-* [DV Style lint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/dv/lint/veriblelint/latest/results.html)
-* [Synthesis results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/syn/latest/results.html)
-* [CDC results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/cdc/latest/results.html)
+* [Chip-level DV simulation results, with coverage (nightly)](https://reports.opentitan.org/hw/top_earlgrey/dv/latest/results.html)
+* [Chip-level Connectivity results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/conn/jaspergold/latest/results.html)
```

Comment:
These also use `chip_earlgrey_asic` as the DUT.

Reply:
Would you suggest to give them all the same rel-path?

Reply:
Yes - I suggest maintaining a uniform path for all too-flows that are run on `chip_earlgrey_asic`.

Reply:
Sounds good, I can align that.

Reply:
I have aligned all chip-level flows to use the same path prefix (`top_earlgrey/ip/chip_earlgrey_asic`).
I am using the `ip` hierarchy since it already exists - but I am happy to set it to something else.